styx-client/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java

```diff
@@ -340,16 +360,16 @@ public void testTokenSuccess() {
   }
 
   @Test
-  public void testTokenFailure() throws Exception {
+  public void testTokenFailure() throws IOException, java.security.GeneralSecurityException {
```

Comment:
We don't need to be so specific I think for test cases, but fine.

Reply:
Agreed, will revert